idaes/core/base/unit_model.py

```diff
@@ -148,7 +148,7 @@ def model_check(blk):
         except AttributeError:
             pass
 
-    def add_port(self, name=None, block=None, doc=None):
+    def add_port(self, name, block, doc=None):
```

Comment:
Slight change in behavior here, but this would already result in an error if these default arguments were used, so I agree with this change.